e2e/playwright/tests/todo_plugin.spec.ts

```diff
@@ -8,35 +8,71 @@
 
 import {expect, test} from '@e2e-support/test_fixture';
 import SlashCommandSuggestions from 'support/components/slash_commands';
-import {fillMessage, getTodoBotDMPageURL} from 'support/utils';
+import {fillMessage, getLastPost, getTodoBotDMPageURL, postMessage, } from 'support/utils';
+
+test.beforeEach(async ({ page, pw }) => {
+  const {adminClient, adminUser} = await pw.getAdminClient();
+  if (adminUser === null) {
+    throw new Error('can not get adminUser');
+  }
+  const dmURL = await getTodoBotDMPageURL(adminClient, '', adminUser.id);
+  await page.goto(dmURL, {waitUntil: 'load'});
+});
 
 export default {
-    connected: () => {
-        test.describe('available commands', () => {
-            test('with just the main command', async ({pages, page, pw}) => {
-               
-            const {adminClient, adminUser} = await pw.getAdminClient();
-            if (adminUser === null) {
-                throw new Error('can not get adminUser');
-            }
-            const dmURL = await getTodoBotDMPageURL(adminClient, '', adminUser.id);
-            await page.goto(dmURL, {waitUntil: 'load'});
-
-            const c = new pages.ChannelsPage(page);
-            const slash = new SlashCommandSuggestions(page.locator('#suggestionList'));
-
-            // # Run incomplete command to trigger help
-            await fillMessage('/todo', page);
-
-            // * Assert suggestions are visible
-            await expect(slash.container).toBeVisible();
-
-            // * Assert help is visible
-            await expect(slash.getItemTitleNth(0)).toHaveText('todo [command]');
-
-            await expect(slash.getItemDescNth(0)).toHaveText('Available commands: list, add, pop, send, settings, help');
-            });
-        });
-    },
+  setup: () => {
+    test('checking available commands', async ({ page }) => {
```

Comment:
```suggestion
  command_autocomplete: () => {
    test('/todo', async ({ page }) => {
```

[0/5](https://handbook.mattermost.com/company/about-mattermost/list-of-terms) on the exact naming to have here, but `setup` makes me think we are setting something up for other tests, which is not really what's going on here